ecosystem/gstreamer_plugin/gst_mtl_common.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2024 Intel Corporation
+ */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "gst_mtl_common.h"
+
+gboolean gst_mtl_common_parse_input_finfo(const GstVideoFormatInfo* finfo,
+                                          enum st_frame_fmt* fmt) {
+  if (finfo->format == GST_VIDEO_FORMAT_v210) {
+    *fmt = ST_FRAME_FMT_V210;
+  } else if (finfo->format == GST_VIDEO_FORMAT_I420_10LE) {
+    *fmt = ST_FRAME_FMT_YUV422PLANAR10LE;
+  } else {
+    return FALSE;
+  }
+
+  return TRUE;
+}
+
+gboolean gst_mtl_common_parse_fps_code(gint fps_code, enum st_fps* fps) {
+  if (!fps) {
+    GST_ERROR("Invalid fps pointer");
+    return FALSE;
+  }
+
+  switch (fps_code) {
```

Comment:
I would sort this cases according to some rule. e.g. alphabetically, or ascending.